std/experimental/allocator/package.d

```diff
@@ -2033,6 +2278,194 @@ class CAllocatorImpl(Allocator, Flag!"indirect" indirect = No.indirect)
     }
 }
 
+/**
+
+Implementation of $(D ISharedAllocator) using $(D Allocator). This adapts a
```

Comment:
For single-word uses of the `D` macro you may want to use the backticks instead.